changelog.dd

```diff
@@ -15,7 +15,13 @@ $(LI $(XREF std_meta, Repeat) was added to obtain a repeating
 $(XREF std_meta, AliasSeq) consisting of template arguments.)
 $(LI $(D fold) was added as an alternative to $(D reduce) with argument order swapped.)
 $(LI $(RELATIVE_LINK2 nextPow2, Added nextPow2 and truncPow2 to std.math.))
-
+$(LI $(REF bsf core,bitop), $(REF bsr core,bitop), and $(REF popcnt, core,bitop) now
+work with ulongs on 32 bit systems.)
+$(LI $(REF topN std,algorithm,sorting) is now 5% - 7% faster.)
+$(LI $(RELATIVE_LINK2 unary, Unary overloads of $(REF startsWith std,algorithm,searching) and
+$(REF endsWith std,algorithm,searching) were added.))
+$(LI $(RELATIVE_LINK2 maxCount, $(REF maxCount std,algorithm,searching) and
+$(REF maxPos std,algorithm,searching) were added.))
```

Comment:
How is the changelog generated? `REF` is in dlang.org.ddoc and invoked correctly here.